xla_native_functions.yaml

```diff
@@ -321,7 +321,6 @@ supported:
   - upsample_nearest2d
   - upsample_nearest2d.vec
```

Comment:
sorry for the late reply - as a first guess, I see that the companion patch in pytorch also makes `upsample_nearest2d.vec a `CompositeImplicitAutograd` op, but you still have the direct lowering for it here. Can you try removing this one too?

If it's important to keep the direct lowering for `upsample_nearest2d.vec`, you'll need to do the same autograd trick that XLA does with `max_pool2/3d`.